### ISSUE TITLE
Mutable and private keywords are swapped 

### ISSUE THREAD

**Issue by panmona** [fsprojects/fantomas#2646]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKAdAdgG0QFzAFsBXPAQwCNcwAHAJwEsA3MvRIgTwFlTLqBeMAFYskADTgoAZ3x4GmAOZTUYANqSIoCNqgASPBxrI0keXnEbtkXS2zFjayzoAMEnTsgBJTEkx4AygwAXshu7hAALE5gALqWAL5hmtHWBkYqpn4W4da29irq4RAAjEnukFxkAB4AMvKINYiKeAAW2UXFAEzO0XE6iZZa5fqGDpBN8ADyAGZ1mIj+BrjtHjZkdg6FRZ1lHgCiPjNzodEQkNjTkL0JZUOraWMUsLDYK1ZrGwWnYADMu1beGR0PAAMXk6wAcogAO7YepvHR4Oj2a79W4pEbpExPF4IvR5TbfCL-M7+GhkaCIABCiGmsDoiAACmQ6GRCPhEHQ8RAkSjwn1tAMdHd3g8MjjXv9cut8mgtuEhCSoGSKdTafSGrBoZzoGQZN4mLBdXJYJhuWBeYhUYL0TlMY9npKMQSvkUwAA2JWQFWUml0hkAVRoRjouv1mENxoYpvN03WMmtECF2hFZ3t4sdeOlnzl3wA7F6fWr-YgAMLYPVSUumqS86B4emx+NW-k3QYYsXYzNSj6yxxugAchfJvvVDK4iEIFE5TewCdbaPbds7UAlWd7hLdAE5h6q-Rrq9gY0q43OW+4BUnbcMV5A1z2Xbm3cVXN9vSPEABBaZsOjVwiEGQ5qWomYDJsky6jBmuIPjKm4dKUb5FvuDL+JODBwEeZons2oHgWAqZ6Le97OnBrodDsSEft+v5oYQGEvMe3wgQuNpLjeUFdjBpE5v2HR-FRqqfnQsDED4AAiiBwvRv7Aci546JeYHXvcnFQGY66Pnx4TFMSb6VFUnjTAAKi0TT+C09J4AA6gw8CtOaPSsVe7GqVi6lZLBvHyu4xSKvp1RGaZTR7GeFlWbZ9ltEq7pORebbCh2amZOYXl9j5OjFJ6AWGZg0wMFUkwhqw9J7FU9CIFIUjRth3wDnFikJSmSXuSlmlkU+HQFjlABKiBwHQ8CRQ5SoRA12hKfhhGpMlGlpfBOlDr1-X0vAELEFOnIzJ4bCEMoSrFHhKmislJDYHIcLzCC9KAXgv7GVB83kTpO5vn1A3wDw50MJdiDXXQt2yV60AtCyFK-gA+tCdkOUdrkna1c08el3zdF6BnCayHCTHQdS1sN0V1eNECTcdaa3kjdpaRl2idIhboVNUmNkNjuMMLW62bXQ227ft3yHc5ynw+Tp3EN9v3-YDnIPekT2deEnSUQzzOs3jeBfRd9SS6wQNviDYP1pyUMw20cOJZBiOecjC3uJ0AkMwZABqcFUvI8DyAoBPmvVZvNRbYyU8M1Oo3pDvVCCYn1jVrs+B7XujcTsRNRBHGW6l1vPbb-lh1U4mwHgADi+BlRVVU1fHRO+ynbkOtxVMddptvZQ7Yua-MVJHtAADWVKst3+BSJMmD+GyZYvBttVuqe87xYu5up7XTr195qPdQzUKwvUNJ4NqTQy4gkn5ZgDAmpgn4+BOXN826LGz2x8819BS9Bw3NMQJ0S3K3CCiYBHmBR6aQIP9WDEAZMZWA3hfDkFPrOGejU55+wXk-dqK83SdFel-X8mBWDME1IoS+046CH3kCfGq18ijTwUhNZOBEWqLxQSjN0PxXwt2+scGobIKDwDIOWWA1VFAb1+rAqhJMaHTXTFxZ+qxg5MPpkUSAJdOQMHZH4dYABpRAiAaCQKaHgbwvcyD-0JlPXCgspp0OQXLRuOgfhK3kR3QxXdjhSGEqJHwUJayIE+q3H69QS4MjLjWOSfI74uQfgjehVi36-HtvYlkKED7s2gIwei2C2DwADMfU0kloAVlZDAnCZ4q60P9pYjO8t3A-FDvIxRKTdHrEWCJYgtZkQ0EWBwZYhS4HUIQdXCJGRA7SNft8H42d5EaK0QZTmhCZgOMwE4+o5CdLOETqTYWRFkokWXowooPxm7yMaRhdWCATgmKKWYyQMQwimCkCCaqKhKEgHiEAA)

#### Code

```fsharp

let mutable private myMutable = 5

```
            

#### Result

```fsharp
let private mutable myMutable = 5

```
            
#### Problem description

As noted in your release description for 5.2.0-alpha-003 I tried the Dallas version out:
Formatting currently leads to invalid F# code when the "mutable" keyword is used, as it swaps 'private' and 'mutable'.

#### Extra information

- [x] The formatted result breaks my code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas Dallas branch

Default Fantomas configuration

**Comment by nojaf** [comment]
Hi @panmona, many thanks for this report! This really helps us a lot!
I'll fix this later today!

**Comment by nojaf** [comment]
Fix available in [v5.2.0-beta-001](https://www.nuget.org/packages/fantomas/5.2.0-beta-001)
